package.json

```diff
@@ -11,9 +11,7 @@
     "i18next": "./bin/cli.js"
   },
   "engines": {
-    "node": "^18.0.0 || ^20.0.0 || ^22.0.0",
-    "npm": ">=6",
-    "yarn": ">=1"
+    "node": "^18 || ^20 || ^22 || ^24"
```

Comment:
We should probably update it to ">=18" so people using uneven version don't have to use workarounds to install the library

Reply:
That depends on the package manager. Some warn, others block installation. Yarn, for example, blocks installation.

Reply:
I see. Then I think `>=18` will be better.